.cicdtemplate/.github/workflows/deploy_staging_and_production_to_firebase_app_distribution.yml

```
@@ -38,6 +39,16 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
+      - name: Set up google-services.json on template-compose
+        env:
+          GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
+          GOOGLE_SERVICES_JSON_STAGING: ${{ secrets.GOOGLE_SERVICES_JSON_STAGING }}
+        run: |
+          mkdir -p template-compose/app/src/production
+          echo $GOOGLE_SERVICES_JSON > template-compose/app/src/production/google-services.json
+          mkdir -p template-compose/app/src/staging
+          echo $GOOGLE_SERVICES_JSON_STAGING > template-compose/app/src/staging/google-services.json
```

As this is the template workflow for a `single` generated project, no `template-compose` folder will be at the root. You can check out some of our last completed work here:
- GA https://github.com/nimblehq/android-templates/blob/develop/.cicdtemplate/.github/workflows/run_detekt_and_unit_tests.yml
- Bitrise: https://github.com/nimblehq/android-templates/blob/feature/448-part2-sample-yaml-file-for-codemagic/.cicdtemplate/.bitrise/bitrise.yml

```suggestion
      - name: Set up google-services.json
        env:
          GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
          GOOGLE_SERVICES_JSON_STAGING: ${{ secrets.GOOGLE_SERVICES_JSON_STAGING }}
        run: |
          mkdir -p app/src/production
          echo $GOOGLE_SERVICES_JSON > app/src/production/google-services.json
          mkdir -p app/src/staging
          echo $GOOGLE_SERVICES_JSON_STAGING > app/src/staging/google-services.json
```

---

All updated in 4c18b2e5